style.css

```diff
@@ -36,13 +36,38 @@ Variables (colors, text, spacing, etc.)
 :root {
 	/* Color Definitions */
 	--color-white: #fff;
+	--color-black: #000;
+	--color-beige: #F5F5DC;
+	--color-navy: #000080;
+	--color-neutral-gray: #ccc;
+	--color-transparent-white: #ffffff2d;
+	--color-slate-grey: #e3e7e882;
+	--color-purple: #e041df;
 
 	/* Color Assignment */
 	--body-background-color: var(--color-white);
 
 	/* Text */
+	/* Font Sizes */
+	--font16: 1rem;
+	--font24: 1.5rem;
+	--font48: 3rem;
+	--font64: 4rem;
+	/* Font Family */
+	--main-heading-font: "Cinzel", serif;
+	--sub-heading-font: "Josefin Sans", sans-serif;
+	--body-font: "Inter", sans-serif;
+	/* Line Height */
 
 	/* Spacing */
+	/* gap */
```

Comment:
Instead of using `--gap` and `--padding`, let's make it generic with `--spacing`. We can then use those `--spacing` variables with gaps, padding, margin, etc. 

Using t-shirt sizing (sm, md, lg, xl) can be useful. Generally, variables should be changeable and you don't want the label to be tied to the value. 

Example: 

`.some-thing {
   padding: var(--spacing-sm);
   margin-bottom: var(--spacing-lg);
}
`

Reply:
Done and done!